leader-election/DistributedMutex/BlobLeaseManager.cs

```diff
@@ -57,23 +55,23 @@ public void ReleaseLease(string leaseId)
                 var leaseClient = this.leaseBlobClient.GetBlobLeaseClient(leaseId);
                 leaseClient.Release();
             }
-            catch (RequestFailedException e) 
+            catch (RequestFailedException e)
             {
                 // Lease will eventually be released.
                 Trace.TraceError(e.ErrorCode);
```

Comment:
Just a quick question (I didn't test), do these trace statements make their way to the console?

Reply:
You'd have to use the dotnet-trace tool to actually get the trace messages. Not trivial, but not as easy as just observing the output window. If we sent this all to output though, it would be pretty noisy. I might just add a mention of the dotnet-trace tool to the README

Reply:
Fair approach.